api/app/Actions/Slack/DeleteSlackNotificationsAction.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\Slack;
+
+use Illuminate\Support\Facades\Auth;
+
+final class DeleteSlackNotificationsAction
+{
+    public function execute()
+    {
+        $user = Auth::user();
+        $user->slack_webhook = null;
+        $user->slack_channel = null;
+        $user->slack_active = null;
```

Comment:
It will not update anything. You need to update these data via repository and then save.